src/app/signup/page.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { describe, expect, it, vi } from "vitest";
```

Comment:
Remove import of describe, expect and it. It is globally recognized

Reply:
I've removed it